### PR TITLE
Remove simulated player zip from runner on startup

### PR DESCRIPTION
**Pull request by tenevdev**
#### Description
I had to apply this fix due to running out of disk space in a test worker node with a game client that was just over 20GB (with a 11GB zip file). I'm not sure if the same applies when launching simulated player deployments (in worker nanny nodes of a separate deployment).

#### Release note
REQUIRED: Add a release note to the `##Unreleased` section of CHANGELOG.md. You can find guidance for writing useful release notes [here](../SpatialGDK/Extras/internal-documentation/how-to-write-good-release-notes.md). Documentation changes are exempt from this requirement.

#### Tests

Tested by adding a command to output disk usage on the runner before and after the remove command. Also searched simulated client logs for previously seen errors about running out of disk space and didn't find any.

#### Documentation
in-code

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Common/LinuxScripts.cs [hunk 1]
```diff
@@ -55,6 +55,13 @@ shift 1
 
         public const string SimulatedPlayerCoordinatorShellScript =
 @"#!/bin/sh
+
+# Some clients are quite large so in order to avoid running out of disk space on the node we attempt to delete the zip
+WORKER_ZIP_DIR=""/tmp/runner_source/""
+if [ -d ""$WORKER_ZIP_DIR"" ]; then
```
Comment (jessicafalk) [inline SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Common/LinuxScripts.cs]:
```suggestion
if [[ -d ""${WORKER_ZIP_DIR}"" ]]; then
```

File: SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Common/LinuxScripts.cs [hunk 2]
```diff
@@ -55,6 +55,13 @@ shift 1
...
+  rm -rf ""$WORKER_ZIP_DIR""
```
Comment (jessicafalk) [inline SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Common/LinuxScripts.cs]:
```suggestion
  rm -rf ""${WORKER_ZIP_DIR}""
```

Comment (tenevdev) [reply SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Common/LinuxScripts.cs]: I don't see a reason to use variable expansion here. Could you justify?

File: SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Common/LinuxScripts.cs (hunk 1 above)
Comment (tenevdev) [reply SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Common/LinuxScripts.cs]: Extended syntax not needed for testing if a directory exists. What I've written is available on all POSIX shells so I prefer to stick with it as long as the condition doesn't require more.

Comment (tenevdev) [reply SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Common/LinuxScripts.cs]: Side note:  this check isn't really required because `rm -rf`

**Pull request merged**